sympy/calculus/util.py

```diff
@@ -38,13 +37,15 @@ def continuous_domain(f, symbol, domain):
     """
     from sympy.solvers.inequalities import solve_univariate_inequality
     from sympy.solvers.solveset import solveset, _has_rational_power
+    from sympy import denom
```

Comment:
Is there any reason not to use the full path to these functions as in the two lines above?

Reply:
I am sorry, I misinterpreted one of the earlier reviews regarding the imports. I am now changing it so as to specify the complete paths, and moving them to top level imports, as recommended by in previous reviews.